### DISCUSSION TITLE
Release v1.5.0

### DISCUSSION THREAD

**Discussion by Antonboom** [Antonboom/testifylint#157] [Q&A] [answered]
@ccoVeille @mmorel-35, hi!

I will implement `encoded-compare` and we will close requirements for https://github.com/Antonboom/testifylint/milestone/1.

How I promised I had added to milestone some patches.
Please, share you opinion what other issues you want to see in the next release.
Let's choose 1-3 patches before focusing on new checkers again.

Thanks

**Comment by ccoVeille** [comment]
OK thanks, I will think about it.

**Comment by mmorel-35** [comment]
What is `encoded-compare` rule ? I might have missed it .

Is  https://github.com/Antonboom/testifylint/pull/153 a patch or a new checker ?

**Reply by mmorel-35** [reply]
`encoded-compare` are JSONEq and YAMLEq , got it 😉

**Reply by Antonboom** [reply]
> a patch or a new checker ?

patch in existing `error-nil` or I missed something?

**Reply by mmorel-35** [reply]
That's also my point of view.

**Comment by Antonboom** [comment, marked as answer]
Milestone is formed.
